DTDL/base_stream.json

```diff
@@ -65,11 +65,6 @@
             "displayName": "Int32",
             "enumValue": "INT32"
           },
-          {
```

Comment:
why did u delete the INT 64 ? 
if you want to set it equal with our stream explorer please change the display name to long